lib/http-bridge/src/Cardano/Wallet/HttpBridge/Compatibility.hs

```diff
@@ -158,6 +161,14 @@ block0 = Block
     , transactions = []
     }
 
+-- | Hard-coded block0 date for Byron
+--
+-- Corresponds to 2017-09-17T04:08:52+00:00.
+-- Retrieved from https://github.com/input-output-hk/cardano-sl/blob/403e26a1d33623f4537517535b7de37773ae9259/scripts/prepare-genesis/README.md#31-genesis-json-format
+-- and also matches 'lib/mainnet-genesis.json' in cardano-sl.
+byronBlock0Date :: UTCTime
+byronBlock0Date = posixSecondsToUTCTime 1505621332
```

Comment:
:+1:

Reply:
Hmmmm.... This depends on the type of network, doesn't it? Also, this time seems wrong for mainnet anyway :s 
See also: 

https://github.com/input-output-hk/cardano-wallet/blob/master/lib/http-bridge/test/bench/Main.hs#L323-L326